### ISSUE TITLE
Update maps with gadgetry

### ISSUE THREAD

**Issue by sprunk** [ZeroK-RTS/Zero-K#3784]
https://github.com/ZeroK-RTS/Zero-K/commit/aeeb21ee397c40e397771dde09f4d73acedb4869
https://github.com/ZeroK-RTS/Zero-K/commit/2bf7ea4347cebd9774bf39dd6315c1aaf5118e78

The first task is to identify maps which have gadgets that use `GetGroundInfo` or `GetTerrainTypeData`. These maps should then be updated to use the rules param to offset the expected return values. Afterwards the wrapping can be removed.

This task is quite unimportant because the wrapping is cheap and isolated, and the risk of mappers using the obsolete interface is low. However a lot can be done without running Spring.

**Comment by GoogleFrog** [comment]
Won't a lot of maps use GetTeamInfo for startbox changes?

**Comment by sprunk** [comment]
No, some maps will use `GetTeamList` to get the team sizes and some will use `Utilities.GetTeamCount` which is ZK-side. Those that do use `GetTeamInfo` will most likely not use customkeys.
